crypto/fipsmodule/aes/asm/aesv8-armx.pl

```diff
@@ -99,6 +99,13 @@
 .Lenc_key:
 ___
 $code.=<<___	if ($flavour =~ /64/);
+#ifdef BORINGSSL_DISPATCH_TEST
+.extern        BORINGSSL_function_hit
+	adrp	x6,:pg_hi21:BORINGSSL_function_hit
+	add x6, x6, :lo12:BORINGSSL_function_hit
+	mov w7, #1
+	strb w7, [x6,#3]
```

Comment:
could you please add a comment stating the name of the flag that's being set (throughout the PR)?